internal/routes/server.go

```diff
@@ -438,6 +438,9 @@ func statsRoutes(r *mux.Router, c *stats.Controller) {
 
 	r.Handle("/realm/key-server.csv", c.HandleKeyServerStats(stats.TypeCSV)).Methods(http.MethodGet)
 	r.Handle("/realm/key-server.json", c.HandleKeyServerStats(stats.TypeJSON)).Methods(http.MethodGet)
+
+	r.Handle("/composite.csv", c.HandleComposite(stats.TypeCSV)).Methods(http.MethodGet)
+	r.Handle("/composite.json", c.HandleComposite(stats.TypeJSON)).Methods(http.MethodGet)
```

Comment:
If we keep the composite API, we need to document it (and its API support level) in docs/api.md.